webpack.config.ts

```diff
@@ -43,7 +44,8 @@ export default (env: Environment, {
 	devServer: {
 		host: 'localhost',
 		port: 3000,
-		open: 'http://localhost:3000',
+		open: os.platform() === 'win32' ? 'explorer' : 'xdg-open',
```

Comment:
While this covers windows and linux use cases, it doesn't work for the mac use case (which is just `open`). So you might need a double ternary there.

Reply:
Pulled the selection of the command string out to a separate function, rather than nested ternary nonsense. I'd originally opted not to handle mac since I had no way of personally validating it, but don't mind fixing it if someone else can!